packages/client/hmi-client/src/temp/ModelEditor.vue

```diff
@@ -708,6 +715,40 @@ const onDownload = () => {
 
 // Entry point
 onMounted(async () => {
+	// Testing
+	mathmlToPetri([
+		'<math display="block" style="display:inline-block;"><mrow><mfrac><mrow><mi>d</mi><mi>S</mi></mrow><mrow><mi>d</mi><mi>t</mi></mrow></mfrac><mo>=</mo><mo>−</mo><mi>β</mi><mi>S</mi><mi>I</mi></mrow></math>',
+		'<math display="block" style="display:inline-block;"><mrow><mfrac><mrow><mi>d</mi><mi>I</mi></mrow><mrow><mi>d</mi><mi>t</mi></mrow></mfrac><mo>=</mo><mi>β</mi><mi>S</mi><mi>I</mi><mo>−</mo><mi>γ</mi><mi>I</mi></mrow></math>',
+		'<math display="block" style="display:inline-block;"><mrow><mfrac><mrow><mi>d</mi><mi>R</mi></mrow><mrow><mi>d</mi><mi>t</mi></mrow></mfrac><mo>=</mo><mi>γ</mi><mi>I</mi></mrow></math>'
+	]).then((data) => {
+		console.log('!!!!!!!!!!!!! MathML to Petri !!!!!!!!!!!!!!!!!!!');
+		console.log(data);
+		console.log('!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!');
+		console.log('');
```

Comment:
You should look into `[console.group()](https://developer.mozilla.org/en-US/docs/Web/API/Console/group)` makes the console tidy and neat

Reply:
neat, didn't know that one